openquakeplatform/bin/oq_install.sh

```diff
@@ -0,0 +1,227 @@
+    cp -rp $INSTALL_DIR/GeoNode*.zip $GEONODE_SHARE
+    cp -rp $INSTALL_DIR/support/geonode.updateip $GEONODE_BIN/geonode-updateip
+    cp -rp $INSTALL_DIR/support/geonode.admin $GEONODE_SHARE/admin.json
+    cp -rp $INSTALL_DIR/support/geonode.local_settings $GEONODE_ETC/local_settings.py
+
+    chmod +x $GEONODE_BIN/geonode
+    chmod +x $GEONODE_BIN/geonode-updateip
+}
+
+function preinstall() {
+    setup_directories
+    reorganize_configuration
+    echo "Fine preinstall"
+}
+
+function randpass() {
+    [ "$2" == "0" ] && CHAR="[:alnum:]" || CHAR="[:graph:]"
+    cat /dev/urandom | tr -cd "$CHAR" | head -c ${1:-32}
+    echo
+}
+
+function setup_postgres_every_time() {
+    true # nothing to do. when we setup DB migrations they'll probably go here.
+}
+
+function setup_django_every_time() {
+    echo "setup_django_every_time:"
+    source ~/env/bin/activate
+    
+    pip -v install /usr/share/geonode/GeoNode-*.zip --no-dependencies --quiet
+    geonodedir=`python -c "import geonode;import os;print os.path.dirname(geonode.__file__)"`
+
+    ln -sf /etc/geonode/local_settings.py $HOME/env/lib/python2.7/site-packages/geonode/local_settings.py
+    ln -sf /usr/lib/python2.7/dist-packages/osgeo $HOME/env/lib/python2.7/site-packages
+
+    # Set up logging symlink
+    mkdir -p $GEONODE_LOG
+    ln -sf /var/log/apache2/error.log $GEONODE_LOG/apache.log
+
+    export DJANGO_SETTINGS_MODULE=geonode.settings
+
+    django-admin migrate account --settings=geonode.settings
+    # geonode runserver &
+    geonode migrate --verbosity 0
+    geonode loaddata $geonodedir/base/fixtures/initial_data.json
+    geonode collectstatic --noinput --verbosity 0
+    geonode createsuperuser
+
+    # Create an empty uploads dir
+    mkdir -p $GEONODE_WWW/uploaded
+    # mkdir -p $GEONODE_WWW/uploaded/thumbs/
+    ln -sf $HOME/env/lib/python2.7/site-packages/geonode/uploaded/thumbs $GEONODE_WWW/uploaded
+    mkdir -p $HOME/env/local/lib/python2.7/site-packages/geonode/uploaded
+    mkdir -p $HOME/env/local/lib/python2.7/site-packages/geonode/uploaded/layers
+    mkdir -p $HOME/env/local/lib/python2.7/site-packages/geonode/uploaded/documents
+
+    # Apply the permissions to the newly created folders.
+    sudo chown www-data -R $GEONODE_WWW
+
+    # Open up the permissions of the media folders so the python
+    # processes like updatelayers and collectstatic can write here
+    chmod 777 -R $GEONODE_WWW/uploaded
+    chmod 777 -R $GEONODE_WWW/static
+    chmod 777 -R $HOME/env/local/lib/python2.7/site-packages/geonode/uploaded/
+    chmod 777 -R $HOME/env/local/lib/python2.7/site-packages/geonode/uploaded/layers
+    chmod 777 -R $HOME/env/local/lib/python2.7/site-packages/geonode/uploaded/documents
+    chmod 777 -R $HOME/env/local/lib/python2.7/site-packages/geonode/static_root
+
+    # for install geonode
+    sudo rm -rf /var/www/geonode/static
+    sudo ln -sf $HOME/env/lib/python2.7/site-packages/geonode/static_root/ /var/www/geonode/static
```

Comment:
Same as above, `static` must not be in `site-packages` and files must be managed by `collectstatic` + `STATIC_URL`